helpers/database-import-scripts/rnacentral/generate_rnacentral_json.py

```diff
@@ -126,9 +180,12 @@ def generate_data_dict(mgnify_accession, sample_accession, taxonomy, deoverlap_d
     global SKIP_CMSCAN, SKIP_GFF, GOOD, BAD_SEQUENCE
     deoverlap_path = os.path.join(deoverlap_dir, "{}.cmscan-deoverlap.tbl".format(mgnify_accession))
     if not os.path.exists(deoverlap_path):
-        logging.warning("cmscan file for accession {} doesn't exist. Skipping.".format(mgnify_accession))
-        SKIP_CMSCAN += 1
-        return None
+        # check if the naming has changed
+        deoverlap_path = os.path.join(deoverlap_dir, "{}.ncrna.deoverlap.tbl".format(mgnify_accession))
+        if not os.path.exists(deoverlap_path):
+            logging.warning("cmscan file for accession {} doesn't exist. Skipping.".format(mgnify_accession))
```

Comment:
Please add the expected path to the log. It is easier to debug if you have the expected path available, especially when something is not supposed to happen.

Reply:
like this:
```python
logging.warning(f"cmscan file for accession {mgnify_accession} doesn't exist. Expected file path {deoverlap_path}. Skipping.")
```

Reply:
Added in https://github.com/EBI-Metagenomics/genomes-catalogue-pipeline/commit/e5acaceef930be1cd356919455fc895693acf87a